src/main/resources/META-INF/plugin.xml

```diff
@@ -3,7 +3,7 @@
   <name>Waifu Motivator</name>
   <vendor email="zaerald.zd@gmail.com" url="https://github.com/zaerald">zaerald</vendor>
 
-  <idea-version since-build="211"/>
+  <idea-version since-build="213.*"/>
```

Comment:
I think you can omit the dot star notation, and just leave it [as the branch number 213](https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html#build-number-format). (Could you also remove the dot star in the gradle properties file as well?)